pages/profiles/ggwilliams.md

```diff
@@ -0,0 +1,27 @@
+# Gabrielle (Gabby) Williams
+
+Eastern Standard Time
+
+Mac OS Sierra
+
+My name is *Gabby* I just finished the General Assembly WDI program in **Boston** where I learned the following languages:
+1. Javascript
+2. Ruby on Rails
+3. Ember
+4. and more...
+
+I hope to learn a lot more through [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io "Open Learning Exchange")
+
+I enjoy:
+*FrontEnd 
```

Comment:
the bullet formatting only shows up when you put a space between "*" and the text.